src/app/pages/NFTInstanceDashboardPage/InstanceImageCard.tsx

```diff
@@ -99,7 +99,8 @@ export const InstanceImageCard: FC<InstanceImageCardProps> = ({ isFetched, isLoa
             minHeight: imageSize,
           }}
         >
-          {isLoading && <Skeleton variant="rectangular" width={imageSize} height={imageSize} />}
+          {/* Has to match imageSize variable */}
+          {isLoading && <Skeleton className="w-[350px] h-[350px]" />}
```

Comment:
Misleading comment?

I think size matching here isn't so important, so doesn't matter. Otherwise inline style would solve this